src/einsteinpy/symbolic/predefined/vacuum_solutions.py

```diff
@@ -66,6 +66,43 @@ def Kerr(c=constants.c, sch=symbols("r_s"), a=symbols("a")):
 
 
 def KerrNewman(
-    c=constants.c, G=constants.G, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
+    c=constants.c, G=constants.G, eps_0=constants.eps_0, sch=symbols("r_s"), a=symbols("a"), Q=symbols("Q")
 ):
-    raise NotImplementedError
+    """
+    Kerr-Newman Metric in Boyer Lindquist coordinates.
+
+    Parameters
+    ----------
+    c : ~sympy.core.basic.Basic or int or float
+        Any value to assign to speed of light. Defaults to ``c``.
+    eps0 : ~sympy.core.basic.Basic or int or float
+        Any value to assign to the electric constant or permittivity of free space. Defaults to ``eps0``.
+    sch : ~sympy.core.basic.Basic or int or float
+        Any value to assign to Schwarzschild Radius of the central object. 
+        Defaults to ``r_s``.
+    a : ~sympy.core.basic.Basic or int or float
+        Spin factor of the heavy body. Usually, given by ``J/(Mc)``,
+        where ``J`` is the angular momentum.
+        Defaults to ``a``.
+    Q:  ~sympy.core.basic.Basic or int or float
+
```

Comment:
Can you complete the docstring for `Q`? It is charge on the black hole.
Write the docstring accordingly.

Reply:
Thanks for catching these, I didn't realize docstring is such a useful thing. I would update them for eps_0, G and Q.